src/components/events/createEvent/basicDetails.tsx

```diff
@@ -6,12 +6,12 @@ import {
   Typography,
 } from '@mui/material';
 import SelectField from '../../general/selectField';
-import DateRangePicker from '../commonEventFields/dateRangePicker';
+import DateRangePicker from '../../general/dateRangePicker';
 import { BLOCKS, EVENT_TYPES } from '../../../types/enums';
 
 interface DetailsProps {
-    currDates: Date[];
-    setCurrDates: (param: Date[]) => void;
+    currDates: string[];
+    setCurrDates: (param: string[]) => void;
     register: UseFormRegister<FieldValues>;
     errors: any;
```

Comment:
errors : ERROR?

Reply:
and using ` FormState<TFieldValues>  {......errors: FieldErrors<TFieldValues>}`
is irrelevant?

Reply:
On the declaration of errors it receives the type `[x: string]: any`.
I don't think it is much of an improvement compared to `any`.